orc8r/gateway/c/common/config/MConfigLoader.cpp

```diff
@@ -113,4 +113,11 @@ bool load_service_mconfig(const std::string& service_name,
   return status.ok();
 }
 
+uint32_t get_log_verbosity_from_mconfig(uint32_t mconfig_log_level) {
```

Comment:
Let's write a small unit test for this just so we have coverage :)

Reply:
Added a small test. :+1: